src/templates/finops-hub/modules/storage.bicep

```
@@ -72,12 +72,25 @@ var schemaFiles = {
   'focuscost_1.0-preview(v1)': loadTextContent('../schemas/focuscost_1.0-preview(v1).json')
   'pricesheet_2023-05-01_ea': loadTextContent('../schemas/pricesheet_2023-05-01_ea.json')
   'pricesheet_2023-05-01_mca': loadTextContent('../schemas/pricesheet_2023-05-01_mca.json')
+  'recommendations_1.0': loadTextContent('../schemas/recommendations_1.0.json')
   'reservationdetails_2023-03-01': loadTextContent('../schemas/reservationdetails_2023-03-01.json')
   'reservationrecommendations_2023-05-01_ea': loadTextContent('../schemas/reservationrecommendations_2023-05-01_ea.json')
   'reservationrecommendations_2023-05-01_mca': loadTextContent('../schemas/reservationrecommendations_2023-05-01_mca.json')
   'reservationtransactions_2023-05-01_ea': loadTextContent('../schemas/reservationtransactions_2023-05-01_ea.json')
   'reservationtransactions_2023-05-01_mca': loadTextContent('../schemas/reservationtransactions_2023-05-01_mca.json')
 }
+var queryFiles = {
+  'HubsRecommendations-AdvisorCost': loadTextContent('./scripts/HubsRecommendations-AdvisorCost.json')
```

Shouldn't these be in the queries folder?
```suggestion
  'HubsRecommendations-AdvisorCost': loadTextContent('./queries/HubsRecommendations-AdvisorCost.json')
```

---

We don't have a `queries` folder. Can I create it at the same level of the `scripts` folder?